astroquery/mast/auth.py

```diff
@@ -0,0 +1,123 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+MAST Authentication
+===================
+
+This file contains functionality related to
+authenticating MAST users.
+"""
+
+import os
+import keyring
+import warnings
+
+from getpass import getpass
+
+from astropy.logger import log
+
+from ..exceptions import AuthenticationWarning
+
+from . import conf
+
+
```

Comment:
Add an `__all__` to all new files, even if it's empty, to keep the namespace cleaner.